### PR TITLE
fix(Kvantum): Kvantum module can recognize ~/Kvantum

### PR DESCRIPTION
**Pull request by cathaysia**

### CODE REVIEW

File: src/plugins/kvantum.py [hunk 1]
```diff
@@ -10,9 +10,13 @@ def available_themes(self) -> dict:
         if not self.available:
             return {}
 
-        path = '/usr/share/Kvantum'
-
-        themes = get_stuff_in_dir(path, type='dir')
+        paths = ['/usr/share/Kvantum',str(Path.home()) + '/.config/Kvantum']
```
Comment (l0drex) [inline src/plugins/kvantum.py:13]: Seems like a whitespace is missing between the two paths in the list

File: src/plugins/kvantum.py [hunk 2]
```diff
@@ -10,9 +10,13 @@ def available_themes(self) -> dict:
...
+        themes = list()
+        # At present, it seems that the function of finding themes is based
+        # on dirs, but .kvconfig. So some theme will no be recognized. This
+        # may be fixed next time
+        for path in paths:
+            themes=themes + get_stuff_in_dir(path, type='dir')
```
Comment (l0drex) [inline src/plugins/kvantum.py:19]: Maybe use `themes.append()` instead

**Review by l0drex — CHANGES_REQUESTED** [review]
Besides the commented things, this looks good :+1:

File: src/plugins/kvantum.py (hunk 2 above)
Comment (cathaysia) [reply src/plugins/kvantum.py:19]:
```python
themes=themes + get_stuff_in_dir(path, type='dir')
```
can work normaly

File: src/plugins/kvantum.py (hunk 1 above)
Comment (cathaysia) [reply src/plugins/kvantum.py]: This will be fixed.

**Review by l0drex — APPROVED** [review]
From my point of view, this can now be merged

**Comment by oskarsh** [comment]
lgtm, nice fix

**Pull request merged**